troposphere/static/js/components/providers/Instances.react.js

```diff
@@ -34,7 +34,10 @@ define(function (require) {
 
     render: function () {
       var provider = this.props.provider,
-        instances = stores.InstanceStore.findWhere({'provider.id': provider.id}),
+        instances = stores.InstanceStore.findWhere({
+            'provider.id': provider.id,
+            'status': 'active'
+        }),
```

Comment:
Because this is a "find" method, I do not believe it is going to do an additional fetch over the wire. We're filtering within the local cache by model _attribute_. 

So you already have all your instances within local cache - and then you perform these "predicate" filters. 

I'll get examples - but _awesome_ question.